es-ds-docs/pages/molecules/carousel.vue

```diff
@@ -0,0 +1,209 @@
+<script setup lang="ts">
+const items = ref([
+    {
+        image_url:
+            'https://edge.disstg.commercecloud.salesforce.com/dw/image/v2/BDDP_STG/on/demandware.static/-/Sites-masterCatalog/default/Products/I-PUPEVCHAR-NEMA.jpg',
+        model_name: 'Wallbox Pulsar Plus 40A',
+        price: '$649.00',
+        productUrl: 'https://equipment.energysage.com/ev-chargers/PUP1-U-1-5-N-002-A',
```

Comment:
Could we use fake data here for the example? We could use something like https://placekitten.com/ to generate images in the sizes we need.